octomap_update:
  ros__parameters:
    input_topic: "/internimage/segmentation/ground_points/acc/filtered_x"
    # voxel cube size for clearing bbox in octomap
    log_each_bbox: true
    bbox_size: 0.5  
    target_frame: "world"
